src/operator/account/CancelAccountOverlay.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {FC, useContext} from 'react'
+import {
```

Comment:
Normally we would alpha-order these for ease of identifying the components, but probably not worth re-running CI for that